_posts/2018-05-24-kotlin-project-setup.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Kotlin Gradle projects and gRPC
+published: false
+permalink: blog/kotlin-gradle-projects
+author: Spencer Fang
+company: Google
+company-link: https://www.google.com
+---
+
+# Kotlin Gradle projects and gRPC
+
+Did you know that gRPC Java now has out of box support for Kotlin projects built with Gradle? Kotlin is a modern statically typed language developed by JetBrains that targets the JVM and Android. It is generally easy for Kotlin programs to interoperate with existing Java libraries. We have added support to the [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin/releases) so that the generated Java libraries are automatically picked up by Kotlin. You can now add the protobuf-gradle-plugin to your Kotlin project, and use gRPC just like you would with a Java or Android project.
+
+Below are Kotlin and Kotlin Android project set up examples.
```

Comment:
optional: "Kotlin and Kotlin" reads oddly, consider rephrasing.

Reply:
Done